### DISCUSSION TITLE
Firefox iOS browser Framework Project

### DISCUSSION THREAD

**Discussion by Saeed118** [mozilla-mobile/firefox-ios#8951] [Ideas]
I have been using this Firefox iOS source code. I'm happy.
But I wanna import this Firefox iOS browser to my another project.
I tried to import, but there are lots of problem.
I wasted lots of time to import, but unfortunately, failed.
If I have FireFox iOS browser Framework , it will help me.
If current Project convert to Framework mode, No problem.
If Firefox iOS browser Framework will be provided, Firefox usage will be increased.
Thanks

**Comment by Koalageddonn** [comment]
The Firefox iOS browser framework project is exciting. It promises to bring a more powerful and customizable browsing experience to iOS users.
